zone/perl_mob.cpp

```diff
@@ -4179,44 +4179,6 @@ XS(XS_Mob_GetResist) {
 	XSRETURN(1);
 }
 
-XS(XS_Mob_GetShieldTarget); /* prototype to pass -Wmissing-prototypes */
```

Comment:
There a reason we're removing calls here? If this is intended we should definitely make a PSA (Changelog) about this function being removed as some folks might be using it somewhere

Reply:
The functions literally did nothing. The old shield variables had no functionality in game.

Reply:
Sounds good